suse_caasp

```diff
@@ -378,20 +382,218 @@ if check_rpm kubernetes-client && [ -f /etc/kubernetes/admin.conf ]; then
     log_cmd $OF 'kubectl version'
     log_cmd $OF 'kubectl api-versions'
     log_cmd $OF 'kubectl config view'
+    log_cmd $OF 'kubectl -n kube-system get pods --field-selector=status.phase=Running'
+    log_cmd $OF 'kubectl get events --sort-by=.metadata.creationTimestamp'
     log_cmd $OF 'kubectl get nodes'
+    log_cmd $OF 'kubectl get all -A'
```

Comment:
this exposes some user workload specific details, i'm not sure if should only stick to the `kube-system` namespace here. not sure how sensitive that is

Reply:
@ereslibre I think you wanted this `-A`. What do you think?

Reply:
I think this should be fine, as it's just a list of everything inside the cluster, without many details about the workloads themselves.